client/components/domains/connect-domain-step/connect-domain-step-login.jsx

```diff
@@ -115,7 +115,10 @@ export default function ConnectDomainStepLogin( {
 									),
 									{
 										em: createElement( 'em' ),
-										a: createElement( 'a', { href: 'https://lookup.icann.org', target: '_blank' } ),
+										a: createElement( 'a', {
+											href: 'https://wordpress.com/site-profiler',
```

Comment:
Will it be better if we wrap it with `localizeUrl` function? You can find it [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/components/domains/connect-domain-step/connect-domain-step-login.jsx#L32)

Reply:
Absolutely, I forgot about it. Thanks!